serialize/Cargo.toml

```diff
@@ -16,6 +16,10 @@ edition = "2018"
 ark-serialize-derive = { path = "../serialize-derive", optional = true }
 ark-std = { git = "https://github.com/arkworks-rs/utils", default-features = false }
 
+[dev-dependencies]
+ark-ff = { path = "../ff" }
```

Comment:
I think you can just use `thread_rng()` from `rand`; no need to import `ark-ff` for this.

Reply:
thread_rng requires std https://docs.rs/crate/rand/0.7.0/source/src/lib.rs

Reply:
How about `StdRng`? Given that we want to clean up this infrastructure in the future, I don't think we should add this dependency here (even if it results in some duplication rn)

Reply:
Done!